### DISCUSSION TITLE
FullWindowOverlay stops working if it loses focus

### DISCUSSION THREAD

**Discussion by Darren120** [software-mansion/react-native-screens#1735] [General]
If a screen loses focus, ie: navigates to another screen, then comes back, any FullWindowOverlay wraps becomes the last in the stack... , maybe because the navigation lib re-ordered everything. 

One way to fix this is to rerender if focus changes ie: focused && <FullWindowOverlay>...</FullWindowOverlay> 

deps: 
...
    "react-native-screens": "~3.18.0",
    "@react-navigation/bottom-tabs": "^6.5.3",
    "@react-navigation/native": "^6.1.1",
    "@react-navigation/stack": "^6.3.11",
...

**Comment by kkafar** [comment]
Hi @Darren120, 
it seems that this is an issue rather than discussion topic. Would you mind filing and issue regarding this with all required informatino?

**Comment by tboba** [comment]
@Darren120 Closing this discussion because of the inactivity, but if you have any questions or problems related to the FullWIndowOverlay, don't hesitate to create a new issue with steps to reproduce (including minimal repro).

Cheers!
